assignments/assignment6.js

```diff
@@ -0,0 +1,74 @@
+//Assignment 6 Chapter 7
+
+//p.266 #8
+
+function reverse(str) {
+    let result = "";
+    for (let i = str.length - 1; i >= 0; i--){
+        result += str.charAt(i);
```

Comment:
This solution is absolutely fine - I just want to call out one point for your future reference: the problem could also have been solved by looping `i` over the string and comparing the `i`th character on the left to the `i`th character on the right and returning false on the first mismatch and returning true after all have matched.  The act of building a string letter by letter is rather slow and in real business/production code is something you might want to avoid.  Comparing the individual letters is a much faster way.   All that said, your solution here is spot-on.